### PR TITLE
feat[WIP]: Product page

### PR DESCRIPTION
**Pull request by saadahmadhere**
This PR focuses on:
1. Add products in the backend.
2. Add Card component for displaying products.
3. Make API calls for products.
4. Display products on the products page.

### CODE REVIEW

File: src/backend/db/products.js [hunk 1]
```diff
@@ -8,23 +8,227 @@ import { v4 as uuid } from "uuid";
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    name: "Boat Air Dopes",
+    image:
+      // https://drive.google.com/uc?export=view&id=132cLoMt9OzjVD9T7em1vkrnJ8Co-uHXW
```
Comment (shivampandey0) [inline src/backend/db/products.js]: remove comment

**Review by shivampandey0 — APPROVED** [review]
Neatly written with individual components.

**Review by abhij1607 — COMMENTED** [review]
Nice work done | | Clean Code | | good use of barrel import

File: src/backend/db/products.js (hunk 1 above)
Comment (saadahmadhere) [reply src/backend/db/products.js]: done

**Pull request merged**